DashAI/front/src/components/experiments/ConfigureModelsStep.jsx

```diff
@@ -157,6 +157,9 @@ ConfigureModelsStep.propTypes = {
     name: PropTypes.string,
     dataset: PropTypes.object,
     task_name: PropTypes.string,
+    input_columns: PropTypes.arrayOf(number),
+    output_columns: PropTypes.arrayOf(number),
```

Comment:
Prefer to use number from proptypes.

```suggestion
    input_columns: PropTypes.arrayOf(PropTypes.number),
    output_columns: PropTypes.arrayOf(PropTypes.number),
```